package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@primer/components",
-  "version": "10.0.1",
+  "version": "11.0.",
```

Looked like you missed the last zero here 😄 
```suggestion
  "version": "11.0.0",
```

---

Oh good golly 😂